apps/web/pages/cart.vue

```diff
@@ -12,9 +12,12 @@
 
 <script setup lang="ts">
 definePageMeta({
-  layoutName: 'checkout',
   pageType: 'static',
 });
 
+const router = useRouter();
 const localePath = useLocalePath();
+const backUrl = localePath(String(router.options.history.state.back));
+const backHref =
+  backUrl === localePath(String(router.options.history.state.current)) ? localePath(paths.home) : backUrl;
```

Comment:
```suggestion
const routerHistoryState = localePath(String(router.options.history.state));
const backUrl = routerHistoryState.back;
const backHref =
  backUrl === routerHistoryState.current ? localePath(paths.home) : backUrl;
```

Reply:
thats not gonna work because ```localePath()``` always returns a string in format ```locale-lang-en/app-base-route```

Reply:
Ah, true, although disappointing. How do you feel about this construction then?
```suggestion
const historyState = router.options.history.state;
const backUrl = localePath(String(historyState.back));
const currentUrl = localePath(String(historyState.current));
const homeUrl = localePath(paths.home);

const backHref = backUrl === currentUrl ? homeUrl : backUrl;
```

Reply:
same as the original, now a bit easier on the eyes 👍 
pushed